.github/workflows/_calculate-docker-image.yml

```diff
@@ -21,11 +21,13 @@ jobs:
     outputs:
       docker-image: ${{ steps.calculate-docker-image.outputs.docker-image }}
     steps:
+      # Use a higher fetch depth to increase the likelihood of having the merge
+      # base
       - name: Checkout PyTorch
         uses: pytorch/pytorch/.github/actions/checkout-pytorch@master
         with:
           submodules: false
-          fetch-depth: 1
+          fetch-depth: 100
```

Comment:
I thought about that, but it feels a bit weird, as it's going to fail for some users. Considering that those jobs are running on our runners, repo checkout is already cached there